### ISSUE TITLE
Supports for charts

### ISSUE THREAD

**Issue by chbndrhnns** [gitpitch/gitpitch#46]
I really like your service and would like to see support for charts, for example by implementing 
https://github.com/rajgoel/reveal.js-plugins.

**Comment by gitpitch** [comment]
Hi @chbndrhnns glad you are enjoying using GitPitch and thanks for your feedback. Having looked at these plugins and recognizing they require both custom mods to the reveal.js configuration and require canvas elements I don't see these as a great fit with a Markdown centric solution like GitPitch.

However, GitPitch does give you the ability to render hi-res images as backgrounds for any slide. This gives you the freedom to use any tool to create the charts you want. Then just screen capture and use the captured images within your GitPitch presentation.

I've created [a sample presentation](https://gitpitch.com/gitpitch/feature-demo/charts) that demonstrates exactly this:

```
https://gitpitch.com/gitpitch/feature-demo/charts
```

This presentation uses charts that were generated by the Chart plugin that you linked to above. Make sure you check out the `PITCHME.md` [for this presentation](https://github.com/gitpitch/feature-demo/blob/charts/PITCHME.md) to see how simple it is to add charts to any presentation.

Check it out and let me know what you think.

**Comment by chbndrhnns** [comment]
Hey there,

Thanks so much for looking into this. Actually I was only about to request the Charts plugin [1] to be included into gitpitch, not the other ones. Sorry for that.

As far as I can see, this plugin does not require changes to the reveal code itself and it just seems attractive for me to be able to have csv data inside of my presentation that is visualized on the fly.
But I cannot tell if it supports Markdown for CSV data, as well.

What do you think?

[1] https://github.com/rajgoel/reveal.js-plugins/tree/master/chart

**Comment by gitpitch** [comment]
What do I think @chbndrhnns ? I think this plugin would make a great new feature for GitPitch. So I've implemented it and the support is now live on gitpitch.com for you to test out.

I've updated the [ sample charts presentation](https://gitpitch.com/gitpitch/feature-demo/charts) so it now renders charts using the Charts.js plugin.

For you to do the same you simply need to add a new property to your `PITCHME.yaml` which will activate this feature, as follows:

```
charts : true
```

Again, check out the raw `PITCHME.md` [for this sample presentation](https://github.com/gitpitch/feature-demo/blob/charts/PITCHME.md) to see an example of how you can now add Chart.js charts to your presentation.

I haven't tried a chart driven by an external `csv` data file yet but as long as you use an absolute URL to your csv data file it should work. If your `csv` data file happens to live in your GitHub repo then you will need to use the absolute URL to the raw file. Relative paths *will not work*.

Let me know how it goes. I won't merge the feature branch for this support into master until I hear back from you about your own testing.

**Comment by chbndrhnns** [comment]
It works amazingly! Thanks very much! I have tried with internal data.

**Comment by gitpitch** [comment]
Excellent, glad you're enjoying this new feature and thanks again for the great suggestion. I've updated the Wiki with a new page for [Chart Slides](https://github.com/gitpitch/gitpitch/wiki/Chart-Slides). Resolved by 805571ae38e6ab437555a2dad0db293f1668ba1a.
